ansible/roles/liquid-core/templates/initialize.d/liquid-core.sh

```diff
@@ -1,3 +1,8 @@
 #!/bin/bash
 
-echo "liquid core was born ready"
+set -e
```

Comment:
You can fold this line into the shebang line: `#!/bin/bash -e`

Reply:
If I fold the `-e`, it'll be the only file with a folded `-e` in the shebang. If we change all of them like that, we'll need to get used to spotting it there, too.

And that will be one more place we need to be watching closely when we're debugging these bash scripts.

Reply:
Yes, I stand corrected, the `set -e` is more readable. Forget that I even mentioned it.